app/src/main/java/org/gdg/frisbee/android/api/PlusPersonDownloader.java

```diff
@@ -51,7 +51,12 @@ public Response intercept(Chain chain) throws IOException {
     }
 
     @Nullable
-    public Person getPersonSync(final String gplusId) {
+    public static Person getPersonSync(final String gplusId) {
```

Comment:
Do we use more than this instance of the gplus client? If yes, probably it would be good to add this to the method name.